site/content/docs/0.1/examples/loading-buttons/index.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: examples
+title: Loading buttons
+extra_css:
+- "loading-buttons.css"
+- "../../assets/css/docs.css"
+extra_js:
+- src: "loading-buttons.js"
+defer: true
+aliases:
+- "/examples/loading-buttons"
+- "/docs/examples/loading-buttons"
+---
+
+<main class="container-fluid container-max-width">
+  <h1 class="py-medium">Loading buttons</h1>
+
+  <p>Find here some examples of how to implement a live loading button, for an indeterminate or determinate time. These examples are not part of the OUDS Web library and must be adapted to your project. You can find the code below.</p>
+
+  <h2 class="py-short">Loading button in an indeterminate time</h2>
+  <p>For this example, the time is set to 5 seconds by Javascript, but in a real project, you just need a callback when the action is finished. An update to the status is done every second to let the user know the download is still ongoing. Each time, a dot is added to the message to make sure it will change and then be vocalized by the screen readers.</p>
+  <div class="row py-taller">
+    <div class="col-4">
+      <button type="button" class="btn btn-default" id="loading-btn-1">
+        Download file 1
```

Comment:
When I test the loading buttons, the animations are broken... 
I would have also add an example of a loading icon button since it is a bit different to manage...  I have a proposal of code refactoring of js to make it work in all cases... To see together.